src/com/jwetherell/algorithms/numbers/Integers.java

```diff
@@ -5,6 +5,12 @@
 import java.util.Map;
 
 public class Integers {
+ // I just moved the values up here, makes the code more "readable"
+    private static final int BILLION = 1000000000;
```

Comment:
The change requested was to make the code more visible by creating constants for power of 10's and put that at the top. To fix this I took the code and put it at the top of the class.